data/week1/2-first-programming-code.md

```diff
@@ -87,14 +90,14 @@ You can check your code by clicking on the `Check` button that is shown in the b
+You are now ready to run your code! To run this program, click on the green triangle has appeared in the margin and click on `run`. You can also use the keyboard shortcut `Ctrl + Shift + F10`. If you followed the above steps, you should now see that the text "Hello World!" is printed to the console, which you can read under the opened `Run` tab at the bottom of your screen. If you want to rerun your code, you can click on the green triangle in the top right corner, or use the keyboard shortcut `Shift + F10`.
 
 <text-box variant="hint" name="Running code">
+  
+Even though running a program is straightforward, a complex process happens in the background. The Java compiler first converts the source code into Java bytecode. Then, the program is executed by a Java interpreter, executing the commands one by one.
 
-Even though running the program is straightforward, a lot is happening behind the scenes. When a program is run, the source code is first compiled into Java bytecode. This compilation process is done by Java compiler, which itself is a program. Following that, the program gets executed, meaning the commands are executed one-by-one by a Java-interpreter that is able to read Java bytecode.
-
-This compile process affects how and when errors occur. When a program is compiled before execution, the compiler can search for errors in it. This also affects the hints provided by the IDE (Integrated Development Environment), and in this way, the programmer can receive immediate feedback on any errors.
+This compile process influences error detection and feedback. The compiler can check for errors before the program is executed, and the IDE (Integrated Development Envirnoment) provides immediate feedback on any issues.
 
-The IDE both compiles and executes the program with just one press of a button. However, the programming environment compiles the program continuously, so it can report errors. You can, for example, try to change the above Hello World exercise print command to `System.out.println("Hello World!")` -- what you'll notice is that the line will be underlined and you'll be notified of an error on the left-hand side.
+The IDE makes the process easy by allowing the programmer to compile and execute the program with a single button press. It also continuously compiles the program, enabling it to report errors. For instance, changing the Hello World exercise print command to `System.out.println("Hello World!")` will result in a line underlining and an error notification on the left side.  
 
 </text-box>
```

Comment:
Aan het einde nog een korte samenvatting van de pagina, zoals we bij de vorige pagina deden?